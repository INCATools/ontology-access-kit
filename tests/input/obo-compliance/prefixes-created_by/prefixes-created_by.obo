format-version: 1.4
ontology: prefixes-created_by
idspace: Y http://example.org/Y/

[Term]
id: X:1
created_by: Y:1

