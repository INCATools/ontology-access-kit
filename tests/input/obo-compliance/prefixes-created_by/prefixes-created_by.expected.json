{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/prefixes-created_by.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "type" : "CLASS",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://www.geneontology.org/formats/oboInOwl#created_by",
          "val" : "Y:1"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#created_by",
      "lbl" : "created by",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ]
  } ]
}